events/ready.js

```diff
@@ -1,6 +1,14 @@
 module.exports = async (client) => {
   console.log(`[API] Logged in as ${client.user.username}`);
-  await client.user.setActivity("Music", {
-    type: "LISTENING",//can be LISTENING, WATCHING, PLAYING, STREAMING
-  });
-};
+let statuses = [

```

Comment:
This Is pretty simple, Its Not API spam and pretty cool (You don't need to add it Not that important!)

Reply:
#102 
Something I asked a LONG time ago after I learned a Lot of Javascript I got a changeable prefix working I also have a reload command to reload commands But I don't really think people need it!